### PR TITLE
Refine the github related setting

### PR DESCRIPTION
**Pull request by xiaoxiang781216**

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -14,9 +14,5 @@ github:
     - mcu
     - microcontroller
   enabled_merge_buttons:
-    # enable squash button:
-    squash:  true
```
Comment (btashton) [inline .asf.yaml:18]: Should we set these to false instead of relying on the default behaviour?

Comment (Ouss4) [reply .asf.yaml:18]: I believe squashing can come in handy.
But if you want to keep it in the contributor's hand, that's okay with me.

Comment (xiaoxiang781216) [reply .asf.yaml:18]: Done.

Comment (xiaoxiang781216) [reply .asf.yaml:18]: > elieve squashing can come in handy.

We can ask contributor to do squash if need. In this way, contributor will learn how to organize the patch series in a better way and apply the knowledge in the furture PR.

**Pull request merged**